gradle-plugins/react/PUBLISHING.md

```diff
@@ -0,0 +1,23 @@
+# Publishing a Signed Plugin
+
+To publish a signed plugin (currently to `mavenLocal` only), you need to configure the signing key data in plugin's `gradle.properties` file
+
+### 1. Add Signing Key Data
+Update `gradle.properties` file with the following properties:
+```
+signing.keyId=<key>
+signing.password=<password>
+signing.secretKeyRingFile=<path-to-gpg-file>
+```
+* `keyId`: The public key ID.
+* `password`: The passphrase used when creating the key.
+* `secretKeyRingFile`: The absolute path to the private key file.
+
+### 2. Publish the Plugin
+Once the signing key is set up correctly, run the following command:
+```sh
+yarn brownfield:plugin:publish:local
+```
+
+### 3. Output
+If everything is configured properly, the signed plugin will be published to the `~/.m2` repository.
```

Comment:
Add info about:
1. clearing m2 before whole process
2. zipping the m2 folder and sending it to URL for maven central portal
3.